addons/gamification/models/res_users.py

```diff
@@ -1,9 +1,17 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import datetime
+
 from odoo import _, api, fields, models
 
 
+class ResUsersLog(models.Model):
+    _inherit = 'res.users.log'
+
+    last_interacted_date = fields.Datetime('Last interacted', default=lambda __: datetime.datetime.now())
```

Comment:
so we do that to have a `last_interacted_date` per login session ? 
(or just to not create a model just for that / lock the user table ?)